src/Esfa.Vacancy.UnitTests/CreateApprenticeship/Application/GivenACreateApprenticeshipRequestValidator/WhenValidatingTrainingCode/WhenValidatingIsTrainingCodeValid.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Esfa.Vacancy.Application.Commands.CreateApprenticeship;
+using Esfa.Vacancy.Application.Commands.CreateApprenticeship.Validators;
+using Esfa.Vacancy.Domain.Entities;
+using Esfa.Vacancy.Domain.Validation;
+using FluentValidation.TestHelper;
+using NUnit.Framework;
+using Ploeh.AutoFixture;
+using Ploeh.AutoFixture.AutoMoq;
+
+namespace Esfa.Vacancy.UnitTests.CreateApprenticeship.Application.GivenACreateApprenticeshipRequestValidator.WhenValidatingTrainingCode
+{
+    [TestFixture]
+    public class WhenValidatingIsTrainingCodeValid
+    {
+        [Test]
+        public void WhenIsTrue_ThenIsValid()
+        {
+            var fixture = new Fixture().Customize(new AutoMoqCustomization());
+
+            var request = fixture.Build<CreateApprenticeshipRequest>()
+                .With(r => r.TrainingType, TrainingType.Standard)
+                .With(r => r.TrainingCode, "123")
+                .With(r => r.IsTrainingCodeValid, true)
+                .Create();
+
+            var validator = fixture.Create<CreateApprenticeshipRequestValidator>();
+
+            validator
+                .ShouldNotHaveValidationErrorFor(req => req.TrainingCode, request);
+        }
+
+        [Test]
+        public void WhenIsFalse_ThenIsValid()
+        {
```

Comment:
Should this method name be WhenIsFalse_ThenIsinValid() ?

Reply:
fixed